docs/README.md

```diff
@@ -62,10 +62,10 @@ Clin supports multiple environments that have to be configured and specified on
 every command.
 
 ## Configuration
-Clin uses configuration using a yaml file located either in the command's
+Clin uses configuration using a YAML file located either in the command's
```

Comment:
Clin uses configuration ~using~ `from` a YAML file...

or

Clin is configured using a Yaml file..